README.md

```diff
@@ -0,0 +1,158 @@
+
+- Ruby version![](https://img.shields.io/badge/Microverse-blueviolet)
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [README](#readme)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello Rails Back End ](#-hello-rails-back-end-)
+  - [Built With ](#built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Open API documentation](#open-api-documentation)
+    - [Usage](#usage-1)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [Show your support ](#show-your-support-)
+  - [📝 License ](#-license-)
+
+# 📖 Hello Rails Back End <a name="about-project"></a>
+
+> This is a back end project to use as a API endpoint for the "Hello world" project. You can find the front-end project here: [Hello Rails Front End](https://github.com/Rachelwebdev/greetings-react).
+
+## Built With <a name="built-with"></a>
+
+- Ruby
+- Rails
+- Postgresql
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[A Hello world page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To run this project, take a copy of the code and follow the instruction below.
+
+### Prerequisites
+
+You need Ruby v3+, Rails v7+ and Postgresql installed on your machine.
+
+### Setup
+
+`Clone the project`
+
+Install gems with:
+
+```
+bundle install
+```
+
+Setup database with:
+
+```
+rails db:create
+rails db:migrate
+```
+
+### Usage
+
+Start server with:
+
+```
+rails server
+```
+
+Visit http://localhost:3000/ in your browser.
+
+### Run tests
+
+Install npm with:
+
+Install rspec with:
+
+```
+bundle install
+```
+
+and
+
+```
+rails generate rspec:install
+```
+
+run the test with:
+
+```
+rspec spec
+```
+
+### Open API documentation
+
+```
+rails server
+```
+
+Visit http://localhost:3000/api-docs in your browser.
+
+### Usage
+
+- Run `rails server` to run the app.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Rachel Isaac**
+
+- GitHub: [@rachelwebdev](https://github.com/Rachelwebdev)
+- Twitter:[@rachelisaac13](https://twitter.com/Rachelisaac13)
+- LinkedIn: [Rachel Isaac](https://www.linkedin.com/in/rachelisaac13/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The LICENSE link is broken, kindly consider adding a license to your project and fix this link so your README is more professional